enclave/core/tracee.c

```diff
@@ -13,13 +12,22 @@
 #include <openenclave/internal/raise.h>
 #include <openenclave/internal/safecrt.h>
 #include <openenclave/internal/safemath.h>
+#include <openenclave/internal/thread.h>
 #include <openenclave/internal/trace.h>
 #include <openenclave/internal/utils.h>
+#include <openenclave/tracee.h>
+
 #include "core_t.h"
+#include "tracee.h"
 
 static oe_log_level_t _active_log_level = OE_LOG_LEVEL_ERROR;
 static char _enclave_filename[OE_MAX_FILENAME_LEN];
 
+const char* const oe_log_level_strings[OE_LOG_LEVEL_MAX] =
```

Comment:
Can this be made static like the variables above? Symbols that are not static are part of OE's binary interface which ought not be expanded unless needed.

Reply:
This can't be made static because we are using the variable that was defined as an extern in https://github.com/openenclave/openenclave/blob/master/include/openenclave/log.h#L20.

I followed the example of host side logging functions - 
https://github.com/openenclave/openenclave/blob/master/host/traceh.c#L32